### PR TITLE
W-13687940-New-content-for-OData-reference-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/apikit-odatav4-extension-reference.adoc [hunk 1]
```diff
@@ -193,6 +193,15 @@ Default configuration
...
 | Strict Search Term Match a| Boolean |  Enables full term match for $search. If enabled, terms in WHERE clause are not enclosed between '%'. |  false | 
+| Sql Limit Rows | Enumeration, one of:
+
+** LIMIT
+** FETCH_NEXT_ROWS_ONLY |  Option to limit the number of rows |  | 
+| Sql Identifiers Delimiter | Enumeration, one of:
+
+** DOUBLE_QUOTES
+** BRACKETS
+** BACK_TICKS |  Option to delimit identifiers |  |
```
Comment (dcourtil) [inline modules/ROOT/pages/apikit-odatav4-extension-reference.adoc]: These list have `**` as a bullet point and the ones in the other document just `*`... Github preview shows these broken as it is trying to make them bold. Is this ok?

**Review by dcourtil — DISMISSED** [review]

File: modules/ROOT/pages/apikit-odatav4-extension-reference.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-odatav4-extension-reference.adoc]: Great catch! Fixed! 
Thanks!

**Review by dcourtil — DISMISSED** [review]

**Review by JuanAller — DISMISSED** [review]

File: modules/ROOT/pages/apikit-odatav4-extension-reference.adoc [hunk 2]
```diff
@@ -193,6 +193,18 @@ Default configuration
 | System Query Options a| Object |  System Query Options map. The grammar and syntax rules for system query options are defined in [OData-ABNF], in order to simplify usability we remove initial dollar sign to avoid escaping withing DataWeave, in example to access System Query Option $skip user should use the following expression #[attributes.systemQueryOptions.'skip'] |  | *x*
 | Search Transformation Enabled a| Boolean |  Enables $search transformation to WHERE clause. |  true | 
 | Strict Search Term Match a| Boolean |  Enables full term match for $search. If enabled, terms in WHERE clause are not enclosed between '%'. |  false | 
+| Sql Limit Rows | Enumeration, one of:
```
Comment (sathya0) [inline modules/ROOT/pages/apikit-odatav4-extension-reference.adoc]:
```suggestion
| Sql Limit Rows a| Enumeration, one of:
```

File: modules/ROOT/pages/apikit-odatav4-extension-reference.adoc [hunk 3]
```diff
@@ -193,6 +193,18 @@ Default configuration
...
+
+* LIMIT
+
+* FETCH_NEXT_ROWS_ONLY |  Option to limit the number of rows |  | 
+| Sql Identifiers Delimiter | Enumeration, one of:
```
Comment (sathya0) [inline modules/ROOT/pages/apikit-odatav4-extension-reference.adoc]:
```suggestion
| Sql Identifiers Delimiter a| Enumeration, one of:
```

File: modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc [hunk 4]
```diff
@@ -150,6 +150,23 @@ As OData shares some similarities with Java database connectivity (JDBC), the `t
 | System query options | Parameter obtained from the OData request using #[attributes.odataRequestAttributes].
 | Search transformation enabled | Parameter to define the behavior of the transformation.
 | Strict search term match | Parameter to define the behavior of the transformation.
+| SQL identifiers delimiter | Option to delimit identifiers in the generated SQL sentence. Available options are DOUBLE_QUOTES, BRACKETS, and BACK_TICKS.
```
Comment (sathya0) [inline modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc]:
```suggestion
| SQL identifiers delimiter a| Option to delimit identifiers in the generated SQL sentence. Available options are DOUBLE_QUOTES, BRACKETS, and BACK_TICKS.
```

File: modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc [hunk 5]
```diff
@@ -150,6 +150,23 @@ As OData shares some similarities with Java database connectivity (JDBC), the `t
...
+
+* DOUBLE_QUOTES: ``SELECT "TrackId", "Name" FROM "Tracks"``
+
+* BRACKETS: ``SELECT [TrackId], [Name] FROM [Tracks]``
+
+* BACK_TICKS: ``SELECT \`TrackId`, \`Name` FROM \`Tracks` ``
+
+| SQL limit rows | Option to limit the numbers of rows. Available options are LIMIT and FETCH_NEXT_ROWS_ONLY.
+
+Examples :
```
Comment (sathya0) [inline modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc]:
```suggestion
Examples:
```

File: modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc [hunk 6]
```diff
@@ -150,6 +150,23 @@ As OData shares some similarities with Java database connectivity (JDBC), the `t
...
+
+Examples:
+
+* DOUBLE_QUOTES: ``SELECT "TrackId", "Name" FROM "Tracks"``
+
+* BRACKETS: ``SELECT [TrackId], [Name] FROM [Tracks]``
+
+* BACK_TICKS: ``SELECT \`TrackId`, \`Name` FROM \`Tracks` ``
+
+| SQL limit rows | Option to limit the numbers of rows. Available options are LIMIT and FETCH_NEXT_ROWS_ONLY.
```
Comment (sathya0) [inline modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc]:
```suggestion
| SQL limit rows a| Option to limit the numbers of rows. Available options are LIMIT and FETCH_NEXT_ROWS_ONLY.
```

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — APPROVED** [review]

**Pull request merged**